specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/virtualMachine.json

```diff
@@ -1953,7 +1953,7 @@
           "description": "Indicates whether failures stemming from the extension will be suppressed (Operational failures such as not connecting to the VM will not be suppressed regardless of this value). The default is false."
         },
         "protectedSettingsFromKeyVault": {
-          "type": "object",
+          "$ref": "./computeRPCommon.json#/definitions/KeyVaultSecretReference",
```

Comment:
was this a mistake in the previous version? this will be considered breaking change. 
will you comment on why this change is being made / justification for breaking change

Reply:
yes, it was a mistake in the previous version. That's being addressed here: https://github.com/Azure/azure-rest-api-specs/pull/20101. It should also get fixed here to avoid confusion